app/assemblers/assemble_exercise_representations_without_feedback.rb

```diff
@@ -0,0 +1,37 @@
+class AssembleExerciseRepresentationsWithoutFeedback
+  include Mandate
+
+  def self.keys = %i[page order criteria track_slug only_mentored_solutions]
+
+  initialize_with :mentor, :params
+
+  def call
+    SerializePaginatedCollection.(
+      representations,
+      serializer: SerializeExerciseRepresentations,
+      serializer_kwargs: { with_feedback: false },
+      meta: {
+        unscoped_total: Exercise::Representation.without_feedback.count
```

Comment:
Scoped to user?

Reply:
(That last bit could be part of https://github.com/exercism/website/pull/2919/files#diff-fe2dd7e4f217bfd3564d47d44f2909ad618f7590f116683124f939bdfc5beeb1R17)